env0/resource_environment_test.go

```diff
@@ -182,6 +184,42 @@ func TestUnitEnvironmentResource(t *testing.T) {
 			})
 		})
 
+		t.Run("Import By Id", func(t *testing.T) {
```

Comment:
This test was missing. Once I added it, the issues described in the bug were revealed (+ additional fields).

Reply:
probably a gap in my understanding, how exactly does this test imitates an import?

Reply:
no problem. Good question!
The following below are used for testing import.

`ImportState:             true,
						ImportStateId:           environment.Id,
						ImportStateVerify:       true,
`